src/layout/shadowdom.ts

```diff
@@ -159,21 +168,24 @@ export class ShadowDom {
   }
 
   public mirrorsRealDom(): boolean {
```

Comment:
Where is the test case for the inconsistent DOM? Wasn't clear from comment or test name.